src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
Please note that as part of the requirement, you are expected to write Tests for your project 👍 

![image](https://github.com/badger-99/space-travelers-hub/assets/20537712/eeed12a8-bb33-4bcd-b28a-916bdd991335)